samples/bluetooth/peripheral_uart/src/main.c

```diff
@@ -463,15 +470,18 @@ void button_changed(uint32_t button_state, uint32_t has_changed)
 		}
 	}
 }
+#endif /* CONFIG_BT_NUS_SECURITY_ENABLED */
 
 static void configure_gpio(void)
 {
 	int err;
 
+#ifdef CONFIG_BT_NUS_SECURITY_ENABLED
```

Comment:
no chance for IS_ENABLED?

Reply:
The sample already uses ifdefs. Using IS_ENABLED may not prevent the issue if we disable compiler optimizations.